workspaces/js/src/types/flow.ts

```diff
@@ -87,6 +87,18 @@ export interface FlowTooltipStep extends CommonStepProps {
    * @defaultValue `false`
    */
   closeOnOverlayClick?: boolean;
+  /**
+   * `z-index` of the tooltip. You should only change this if this tooltip is positioned differently than your default.
+   *
+   * Defaults to `var(--flows-zIndex)` from the style template.
+   */
+  zIndex?: string;
```

Comment:
New options to change `z-index` of tooltip and modal and when using `overlay: true` also target element z-index. All the options are per step.

Do you think I've missed something? @OPesicka